cinema/views.py

```diff
@@ -116,7 +116,7 @@ def get_serializer_class(self):
     def upload_image(self, request, pk=None):
         movie = self.get_object()
         serializer = self.get_serializer(movie, data=request.data)
-        if serializer.is_valid():
+        if serializer.is_valid(raise_exception=True):
```

Comment:
don't use `if` and response with 400 status if you use raise_exception=True, because it do it automatically